lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -260,14 +265,19 @@ def commit(method, endpoint, parameters, options = {})
         url = "#{(test? ? test_url : live_url)}/#{endpoint}"
 
         begin
-          raw_response = ssl_request(method, url, post_data(parameters), headers(options))
+          raw_response = if options[:action] == 'inquire'
+                           ssl_get(url, headers(options))
+                         else
+                           ssl_request(method, url, post_data(parameters), headers(options))
+                         end
+
```

Comment:
I tried that but would get a error from Decidir because they don't accept any parameters for this endpoint. `post_data` will return an empty hash as default

Reply:
Interesting! Decidir doesn't just ignore the empty `post_data` hash in the `GET` request? 😳

Reply:
Nope, I keep getting `ArgumentError: GET requests do not support a request body`. This is probably because `post_data` convert the empty hash into a json so it probably sends as `"{}"`

Reply:
Try setting the `parameters` to `nil` when you call `commit` from your new method and then in the `post_data` method, just return if it's `nil`.

Reply:
I got it working & will push the new changes. Apparently `void` wants json even if it's an empty hash